### PR TITLE
Support codespell

### LINKED ISSUE #141: CI: support codespell
See https://github.com/codespell-project/codespell. This not only checks comments in code, but also documentation for spelling mistakes.

Should create a circleci configuration with a "spelling" task running codespell on the repo.

Also while doing so, codespell should detect typos which need to be corrected.

This should be an easy task.

### PR DESCRIPTION
**Pull request by bakaoh**
and correct typos

Fix #141

### CODE REVIEW

**Comment by bakaoh** [comment]
@axic -- i updated as your request, although i still prefer run-list to multi-line command

**Comment by vs77bb** [comment]
@axic I think this one might be with you for review, still? 🙂

**Comment by axic** [comment]
I wonder why this hasn't caught the typos from #139. Though mostly only `large` vs.` lage` could have been caught.

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,8 +6,11 @@ jobs:
 
     steps:
     - checkout
-    - run: sudo pip install codespell
-    - run: codespell --skip=".git,./assets"
+    - run:
```
Comment (axic) [inline .circleci/config.yml]: Please do not include this CI change in the `Fixed typos` commit. Can you please move it into the first commit?

Comment (bakaoh) [reply .circleci/config.yml]: @axic --- i updated as your request :)

**Comment by bakaoh** [comment]
> I wonder why this hasn't caught the typos from #139. Though mostly only `large` vs.`lage` could have been caught.

It is because `lage` is not in codespell dictionary file https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary.txt. I can add a custom dictionary for this word but it's solve nothing. It's better to improve codespell itself to contain more common misspellings. Also because the way they do it (by using a dictionary file), i don't think we should expect that will catch all typos

**Pull request merged**